artifactory/services/deleteRepository.go

```diff
@@ -0,0 +1,40 @@
+package services
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-client-go/artifactory/auth"
+	rthttpclient "github.com/jfrog/jfrog-client-go/artifactory/httpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+type DeleteRepositoryService struct {
+	client     *rthttpclient.ArtifactoryHttpClient
+	ArtDetails auth.ArtifactoryDetails
+}
+
+func NewDeleteRepositoryService(client *rthttpclient.ArtifactoryHttpClient) *DeleteRepositoryService {
+	return &DeleteRepositoryService{client: client}
+}
+
+func (drs *DeleteRepositoryService) GetJfrogHttpClient() *rthttpclient.ArtifactoryHttpClient {
+	return drs.client
+}
+
+func (drs *DeleteRepositoryService) Delete(repoKey string) error {
+	httpClientsDetails := drs.ArtDetails.CreateHttpClientDetails()
+	log.Info("Deleting repository......")
```

Comment:
...... --> ...
(Our standard is three dots).
This comments applies to a few other messages in the code.